neuro_san/service/http/server/http_server.py

```diff
@@ -128,11 +129,29 @@ def __call__(self, other_server: AgentServer):
                          self.server_config.http_idle_connection_timeout_seconds)
         self.logger.info({}, "HTTP server is shutting down after %d requests", self.requests_limit)
 
+        if self.server_config.http_server_monitor_interval_seconds > 0:
+            # Start periodic logging of server resources used:
+            tornado.ioloop.PeriodicCallback(
+                self.log_resources_usage,
+                self.server_config.http_server_monitor_interval_seconds * 1000).start()
+
         tornado.ioloop.IOLoop.current().start()
         self.logger.info({}, "Http server stopped.")
         if other_server is not None:
             other_server.stop()
 
+    def log_resources_usage(self):
+        """
+        Log current usage of server run-time resources:
+        file descriptors and open inet connections on server port.
+        """
+        # Get used file descriptors:
+        fds, soft_limit, hard_limit = ServiceResources.get_fd_usage()
+        # Get active TCP connections to our http port:
+        conn = ServiceResources.active_tcp_on_port(self.http_port)
+        self.logger.info({}, "Used: file descriptors %d (%d, %d) connections: %d",
+                         fds, soft_limit, hard_limit, conn)
+
```

Comment:
Query ServiceResources class for things we are interested in:
file descriptors + open connections.
Log that info.